x-pack/legacy/plugins/siem/public/components/header_page/editable_title.tsx

```diff
@@ -43,10 +48,12 @@ const EditableTitleComponent: React.FC<Props> = ({ onSubmit, isLoading, title })
 
   const onClickSubmit = useCallback(
     (newTitle: string): void => {
-      onSubmit(newTitle);
```

Comment:
Not associated with what you did but I think that will be cleaner
```
  const [editMode, setEditMode] = useState(false);
  const [changedTitle, onTitleChange] = useState<string>(typeof title === 'string' ? title : '');

  const onCancel = useCallback(() => setEditMode(false), []);
  const onClickEditIcon = useCallback(() => setEditMode(true), []);

  const onClickSubmit = useCallback((): void => {
    if (changedTitle !== title) {
      onSubmit(changedTitle);
    }
    setEditMode(false);
  }, [changedTitle, title]);

  const handleOnChange = useCallback(
    (e: ChangeEvent<HTMLInputElement>) => {
      onTitleChange(e.target.value);
    },
    [onTitleChange]
  );

  return editMode ? (
    <EuiFlexGroup alignItems="center" gutterSize="m" justifyContent="spaceBetween">
      <EuiFlexItem grow={false}>
        <EuiFieldText
          onChange={handleOnChange}
          value={`${changedTitle}`}
          data-test-subj="editable-title-input-field"
        />
      </EuiFlexItem>
      <EuiFlexGroup gutterSize="none" responsive={false} wrap={true}>
        <EuiFlexItem grow={false}>
          <EuiButton
            color="secondary"
            data-test-subj="editable-title-submit-btn"
            fill
            iconType="save"
            onClick={onClickSubmit}
            size="s"
          >
            {i18n.SAVE}
          </EuiButton>
        </EuiFlexItem>
...
```

Reply:
done ;)